rfc/20240611-ExcludeFlag.md

```diff
@@ -0,0 +1,59 @@
+# Add Exclude Flag for Targeted Plan/Apply
+
+Issue: https://github.com/opentofu/opentofu/issues/426
+
+This RFC details a new flag, `-exclude`, which would do the inverse of `-target` - that is, plan/apply all resources except those specified by the exclude flag and their descendants.
+
+Only for use in exceptional circumstances as is the case for `-target`, this makes it a lot easier to plan/apply all changes to an environment except ones with known issues eg where there is a problem with the infrastructure or the committed config for one module/resource, but you still want to apply everything else.
+
+https://github.com/hashicorp/terraform/issues/2253 is one of the most requested features in the Terraform repo
+
+
+## Proposed Solution
```

Comment:
This section is missing the specifics of how the `-exclude` flag will work. For example - what happens to dependencies and dependents of an excluded resource, what happens when there's a mixture of `-target` and `-exclude` flag in the same execution (should not be allowed for now)

For example - following up on [this comment](https://github.com/opentofu/opentofu/issues/426#issuecomment-2137148391) from the issue